### DISCUSSION TITLE
crackstatus

### DISCUSSION THREAD

**Discussion by inkstop1** [iMAboud/iMSteam#3] [General]
add pls

const new3SiteSearchLink = `https://crackstatus.net/tracker.php?nm=${formattedGameName}`;
  createButton(new3SiteSearchLink, "Crackstatus", "Search on Crackstatus", "https://crackstatus.net/styles/templates/default/images/Hot_icons/rel/PNL2.png");

**Comment by iMAboud** [comment]
I've never heard of this site, is it safe ? 
Maybe I'll add a feature that allows anyone to add any site they like and hide what they don't like.

**Reply by inkstop1** [reply]
yea, it safe, i'm one of the admins of this resource.
![pic](https://github.com/iMAboud/iMSteam/assets/155008923/e850bb3c-b2d1-4dbb-b090-7bfd76451e73)
we dug into the script code and added several sites for ourselves

**Reply by iMAboud** [reply]
Done, I updated the extension and added crack status and a few adjustments
